src/parkapi_sources/converters/base_converter/push/normalized_xlsx_converter.py

```diff
@@ -73,7 +73,9 @@ def handle_xlsx(self, workbook: Workbook) -> tuple[list[StaticParkingSiteInput],
             # ignore empty lines as LibreOffice sometimes adds empty rows at the end of a file
             if row[0].value is None:
                 continue
-            parking_site_dict = self.map_row_to_parking_site_dict(mapping, row)
+            parking_site_dict = self.map_row_to_parking_site_dict(
+                mapping, row, **{'column_names': [cell.value for cell in next(worksheet.rows)]}
```

Comment:
Please change this to one parameter per line. To clearify what you do with more arguments, it's always a good idea to use keyword arguments.

Reply:
Do you mean formatting and placing ```column_names``` on a new line? I have done just that.

Reply:
Does it still need further modifications

Reply:
No, more like this:

```
 parking_site_dict = self.map_row_to_parking_site_dict(
    mapping=mapping, 
    row=row, 
    column_names=[cell.value for cell in next(worksheet.rows)],
```

Then it's much easier to see what's happening there.

Reply:
Ok i tried adjusting it to one parameter per line but unfortunately it does not pass the github lint test. I had to use ```ruff format ./src ./tests``` reformat the parameters again. I hope that's okay leaving all parameters on one line.